samples/servicecontrol/adapter-sqlserver/ServiceControlAdapter/SqlServerTransportAdapter/Sales/Database.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading.Tasks;
+
+class Database
```

Comment:
Hmm the purpose of it is to generate failures only so how about renaming it to failure simulator?